From 5f005830eea7d03c02107a3a3fc58907b0a037bf Mon Sep 17 00:00:00 2001
From: Alejandro del Castillo <alejandro.delcastillo@ni.com>
Date: Mon, 8 Apr 2019 11:14:56 -0500
Subject: [PATCH] regress/issue72.py: resolve paths before comparision

In systems that have a volatile /tmp, the test incorrectly fails since
it doesn't resolve the real path in all cases.

Signed-off-by: Alejandro del Castillo <alejandro.delcastillo@ni.com>
---
Upstream-Status: Submitted

 tests/regress/issue72.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/regress/issue72.py b/tests/regress/issue72.py
index 1626877..2f24dec 100755
--- a/tests/regress/issue72.py
+++ b/tests/regress/issue72.py
@@ -56,7 +56,7 @@ if not os.path.lexists("{}/{}".format(cfg.offline_root, long_filename2)):
 				"not created.")
 
 linky = os.path.realpath("{}/{}".format(cfg.offline_root, long_filename2))
-linky_dst = "{}/{}".format(cfg.offline_root, long_filename)
+linky_dst = os.path.realpath("{}/{}".format(cfg.offline_root, long_filename))
 if linky != linky_dst:
 	opk.fail("symlink path truncated.")
 
-- 
2.20.1

